tests/Nodes/Embedded/SVGImageElementTest.php

```diff
@@ -1,11 +1,13 @@
 <?php
 
+namespace SVG;
```

Comment:
Adding namespace declarations for the tests is a good idea. But then I think they should be the same as for the class under test. For example in this case, `SVG\Nodes\Embedded`.